src/GraphQL.Tests/Utilities/CustomGraphQLAttributeTests.cs

```diff
@@ -56,5 +88,30 @@ public Post GetPostById(string id)
         {
             return PostData.Posts.FirstOrDefault(x => x.Id == id);
         }
+
+        public ABlog Blog(string id)
+        {
+            return new ABlog();
+        }
+    }
+
+    [GraphQLMetadata("IUniqueElement")]
+    public abstract class UniqueElement
+    {
+        public abstract string Id { get; }
+    }
+
+    [GraphQLMetadata(IsTypeOf = typeof(ABlog))]
```

Comment:
Looking at this it does seem a bit redundant.  Perhaps we should provide a default implementation in the `TypeConfig`, which you can override with this `IsTypeOf` property you added to the metadata?  `TypeConfig` contains the `Type`, in this case `ABlog`, already.

Reply:
You're right. Let me see how I can provide the default implementation...